frontend/src/modules/entitieslist/components/EntitiesList.tsx

```diff
@@ -116,6 +123,29 @@ export class EntitiesListBase extends React.Component<InternalProps> {
                 });
             }
         }
+
+        const currEntities = this.props.entities.entities;
+        const prevEntities = prevProps.entities.entities;
+
+        if (prevEntities.length !== 0) {
+            if (prevEntities !== currEntities) {
+                let PK = [];
+                prevEntities.forEach((e) => {
+                    PK.push(e.pk);
+                });
+
+                let siblingPKs = [];
+                currEntities.filter((e) => {
+                    if (!PK.includes(e.pk)) {
+                        siblingPKs.push(e.pk);
+                    }
+                });
+
+                this.setState({
+                    siblingIds: siblingPKs,
+                });
+            }
+        }
```

Comment:
A more elegant solution would be to set a new `sibling` flag in the `map_entities` function in Djangoon the when siblings are returned.

Reply:
To clarify this solution, is it to let Django handle all the filtering with the new `sibling` flag?

Reply:
April and I just solved this it was way more elegant 😄